packages/tools/components-package/nps.js

```diff
@@ -56,8 +57,9 @@ const getScripts = (options) => {
 				json: `node "${LIB}/i18n/toJSON.js" src/i18n dist/generated/assets/i18n`,
 			},
 			jsonImports: {
-				default: "mkdirp dist/generated/json-imports && nps build.jsonImports.themes build.jsonImports.i18n",
+				default: "mkdirp dist/generated/json-imports && nps build.jsonImports.themes build.jsonImports.i18n build.jsonImports.illustrations",
```

Comment:
"build.jsonImports.illustrations", as we refactored the path name to js-import, it should be also moved to separate task - "build.jsImports.illustrations""

Reply:
Done.